src/qt/walletmodel.h

```diff
@@ -155,6 +155,9 @@ class WalletModel : public QObject
     AddressTableModel* getAddressTableModel() const { return addressTableModel; }
 
     void refresh(bool pk_hash_only = false);
+
+    uint256 getLastBlockProcessed();
```

Comment:
cd3b6d0666fe0e6191596e2aa56980ab60841f2d

Can be const.

Reply:
> Can be const.

Correct.

Reply:
[Updated](https://github.com/bitcoin-core/gui/pull/8#issuecomment-647760931).